package.json

```diff
@@ -59,9 +65,16 @@
       "@types/express": "^4.17.4",
       "@types/jest": "^25.2.1",
       "@types/node": "^13.11.0",
+      "@types/node-cron": "^2.0.3",
       "@types/uuid": "^7.0.2",
+      "@typescript-eslint/eslint-plugin": "^2.19.0",
+      "@typescript-eslint/parser": "^2.19.0",
       "babel-jest": "^25.1.0",
       "chai": "^4.2.0",
```

Comment:
I guess this can be removed?

Reply:
+ chai as well?

Reply:
I'll remove those. As for `chai`, I've used its `assert` so I have to replace it with `Jest`s  `expect`.

Reply:
removed `chai` as well 👍